maintainers/maintainer-list.nix

```diff
@@ -3804,6 +3804,13 @@
     githubId = 98980;
     name = "Chmouel Boudjnah";
   };
+  chn = {
```

Comment:
Update to the `maintainer-list` does not follow the [commit convention](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions):

> When adding yourself as maintainer in the same pull request, make a separate commit with the message maintainers: add <handle>. Add the commit before those making changes to the package or module. See [Nixpkgs Maintainers](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) for details.

While you're at it, you can also make separate commits for formatting and adding maintainers to the package as you should:

> Create a commit for each logical unit.

Which makes things clearer.

Reply:
Thank you for pointing out this. I will correct it tomorrow.

Reply:
I'm pretty sure the convention should be `maintainers: add chn` but I could be wrong, please correct me @eljamm

Reply:
> I'm pretty sure the convention should be maintainers: add chn

Yes, that's correct.

Reply:
I will correct it now.

Reply:
Thank you :heart:

Reply:
Also, following the [versioning guide](https://github.com/NixOS/nixpkgs/blob/0a0c9e56cd7cfe1b18745fd5c6c741a3d1b6a0ca/pkgs/README.md#versioning), I think the package version and commit title should be changed from `0.0.18` to `"0.0.18-unstable-2024-09-22"` (and also remove `with patches in master` from the commit title)